exec/continuous.stan

```diff
@@ -62,9 +62,59 @@ functions {
   }
   
   /** 
-  * Pointwise (pw) log-likelihood vector
+  * Apply inverse link function to linear predictor for beta models
   *
-  * @param y The integer array corresponding to the outcome variable.
+  * @param eta Linear predictor vector
+  * @param link An integer indicating the link function
+  * @return A vector, i.e. inverse-link(eta)
+  */
+  vector linkinv_beta(vector eta, int link) {
+    vector[rows(eta)] mu;
+    if (link < 1 || link > 6) reject("Invalid link");
+    if (link == 1)  // logit
+      for(n in 1:rows(eta)) mu[n] = inv_logit(eta[n]);
+    else if (link == 2)  // probit
+      for(n in 1:rows(eta)) mu[n] = Phi(eta[n]);
+    else if (link == 3)  // cloglog
+      for(n in 1:rows(eta)) mu[n] = inv_cloglog(eta[n]);
+    else if (link == 4) // cauchy
+      for(n in 1:rows(eta)) mu[n] = cauchy_cdf(eta[n], 0.0, 1.0);
+    else if (link == 5)  // log 
+      for(n in 1:rows(eta)) mu[n] = exp(eta[n]);
+    else if (link == 6) // loglog
+      for(n in 1:rows(eta)) mu[n] = 1-inv_cloglog(-eta[n]); 
+      
+    for (n in 1:rows(mu)) { 
+      //FIXME: maybe check this in tests but not in released version?
+      if (mu[n] < 0 || mu[n] > 1)
+        reject("mu needs to be between 0 and 1")
```

Comment:
And I guess it could just be done inside the same loop that defines `mu` instead of a separate one